spec/models/user_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'validations for User model' do
+    before(:each) do
+      @user = User.new(name: 'Tom', photo: 'image.png', bio: 'Teacher from Mexico', post_counter: 0)
+    end
+
+    before { @user.save }
+
+    it 'if there is name' do
```

Comment:
- [ ] I will encourage you to doeble-check the tests written again, even when I ran the test one by one, they still all did not pass.

### Error message
![image](https://github.com/AlyAtalla/blog-app/assets/86200356/77d2325d-5b7e-4179-b7c2-c0a1bda8cf44)